### DISCUSSION TITLE
CompreFace 1.0.0

### DISCUSSION THREAD

**Discussion by pospielov** [exadel-inc/CompreFace#770] [General]
## Features

* Manage Collection page. Now you can manage subjects and images with our user-friendly UI. 
* Added 'Delete images in bulk' endpoint. Now you can delete several examples at one request.
* Added mask plugin for Detection and Verification services
* Added default Detection and Verification face services
* Added query parameter to "List of All Saved Examples of the Subject" endpoint. Now you can filter examples by subject. 

## Fixes

* https://github.com/exadel-inc/CompreFace/issues/556
* https://github.com/exadel-inc/CompreFace/issues/723
* Other fixes

## More updates

* We released [Postman documentation](https://documenter.getpostman.com/view/17578263/UUxzAnde)
* [Instructions](https://github.com/exadel-inc/compreface-kubernetes#deployment-on-aws) on how to install CompreFace on AWS and CloudFormation script
* Released [single docker image](https://github.com/exadel-inc/CompreFace/blob/master/docs/Installation-options.md) option to deploy CompreFace

## Migration from 0.6.1 version
* No special migration is needed

## How to install:
1. Download CompreFace_1.0.0.zip archive or run:
`wget -q -O tmp.zip 'https://github.com/exadel-inc/CompreFace/releases/download/v1.0.0/CompreFace_1.0.0.zip' && unzip tmp.zip && rm tmp.zip`
2. To start CompreFace run:
`docker-compose up -d`
3. Open in your browser: http://localhost:8000/login
4. To stop properly run: 
`docker-compose stop`
5. To start again run: 
`docker-compose start`

More detail install guide(and Windows instructions) is on our [readme](https://github.com/exadel-inc/CompreFace#getting-started-with-compreface)﻿ page.

<hr /><em>This discussion was created from the release <a href='https://github.com/exadel-inc/CompreFace/releases/tag/'>CompreFace 1.0.0</a>.</em>

**Comment by martinenkoEduard** [comment]
Is there a discord channel or something?
What is the maim place to ask questions about CompreFace?

**Reply by pospielov** [reply]
The main place is this repository. 
You can also ask in [Gitter](https://gitter.im/CompreFace/community), as notifications from it receive all community members, there are more chances that you receive the answer from anybody else.
